web/webkit/src/main/scala/net/liftweb/http/js/LiftJavaScript.scala

```diff
@@ -75,20 +75,14 @@ object LiftJavaScript {
   }
 
   def initCmd(settings: JsObj): JsCmd = {
-    val extendCmd = LiftRules.jsArtifacts match {
-      case jsa: JQueryArtifacts => Call("window.lift.extend", JsVar("lift_settings"), JsVar("window", "liftJQuery"))
+    val extendJqOrVanilla = LiftRules.jsArtifacts match {
```

Comment:
Why such a specific name? Currently these are the only two things this can extend, but there's no real reason to capture that in the variable name IMO. Perhaps `extendJsHelpersCmd`?